src/Catalyst.Core.Modules.Rpc.Server/RpcServerModule.cs

```diff
@@ -40,6 +40,11 @@ protected override void Load(ContainerBuilder builder)
 
             async void BuildCallback(IContainer container)
             {
+                if (container == null)
+                {
+                    throw new ArgumentNullException(nameof(container));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The parameter name '' is not declared in the argument list.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4416347)